src/proguard/proguard.targets

```diff
@@ -2,7 +2,7 @@
   <UsingTask AssemblyFile="$(MSBuildThisFileDirectory)..\..\bin\Build$(Configuration)\Xamarin.Android.Tools.BootstrapTasks.dll"  TaskName="Xamarin.Android.Tools.BootstrapTasks.Ant" />
   <Target Name="_BuildProGuard">
     <Ant
-        Arguments="-verbose -f buildscripts\build.xml proguard"
+        Arguments="-f buildscripts\build.xml proguard"
```

Comment:
alternatively we could just use `$(ProguardAdditionalArgs)` in place of the `-verbose`. That way people could just add it to the Configuration.override.props

Reply:
That should probably be actually `<AntProguardBuildAdditionalArguments>`; it's not about proguard arguments.